README.md

```diff
@@ -334,9 +334,11 @@ php artisan policy:add writer,articles,edit
 Adds a role for a user:
 
 ```bash
-php artisan role:assign eve writer
+php artisan role:assign eve writer --type=g2

```

Comment:
Please use `ptype` instead of `type` to ensure consistency with the code.
`--ptype` is not required, so two usages should be retained.